tests/acceptance/features/user/public_user/create_account.feature

```diff
@@ -0,0 +1,15 @@
+Feature: Create new account
+
+    @public-user @user @add-user
+    Scenario: Creating new account
+        Given I am on the homepage
```

Comment:
There should be a check to see if the create account feature is enabled and skip the test if it isn't (otherwise the test will fail when it doesn't see the "Create User Account" text).